src/ActiveLogin.Identity.Swedish.TestData/TestData.fs

```diff
@@ -0,0 +1,49 @@
+namespace ActiveLogin.Identity.Swedish.FSharp.TestData
+open ActiveLogin.Identity.Swedish.FSharp
+open ActiveLogin.Identity.Swedish.TestData.AllPins
+open System
+
+
+module SwedishPersonalIdentityNumberTestData =
```

Comment:
Nice work on this one! I'd love to have a module that exposes the same functions, but in a "C#" way, the same we did with the core  package. Like different casing etc.